packages/suite/src/components/suite/CoinBalance.tsx

```diff
@@ -4,13 +4,15 @@ import { FormattedCryptoAmount } from 'src/components/suite';
 interface CoinBalanceProps {
     value: string;
     symbol: Account['symbol'] | (string & {});
+    justifyContent?: 'flex-end' | 'flex-start';
```

Comment:
`FlexJustifyContent` would maybe make more sense here?

Reply:
Well I don't need other options in this component

Reply:
so you can pick those two from `FlexJustifyContent`?

Reply:
sure

Reply:
This is not there anymore